go/l1client/txhandler/handler.go

```diff
@@ -0,0 +1,168 @@
+package txhandler
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/obscuronet/obscuro-playground/go/buildhelper/buildconstants"
+	"github.com/obscuronet/obscuro-playground/go/log"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/nodecommon"
+)
+
+const methodBytesLen = 4
+
+type TxHandler interface {
```

Comment:
We need to come up with a better name for this. Any ideas @joeldudley , @otherview ?

Reply:
Will take sugestions! It's not descriptive at all